larq_compute_engine/mlir/python/graphdef_tfl_flatbuffer.cc

```diff
@@ -80,6 +81,11 @@ pybind11::bytes ConvertGraphDefToTFLiteFlatBuffer(
       // which default to mean=0.0 and std=1.0.
       quant_specs.input_ranges.push_back({-128.0, 127.0});
     }
+    if (!default_ranges.is_none()) {
+      quant_specs.inference_input_type = tensorflow::DT_QINT8;
```

Comment:
Setting `inference_input_type=int8` will get rid of the need for Quantize / Dequantize ops at beginning and end of the network.

Reply:
Thats great because they were blowing up the MIAM for some models.

Reply:
So now this is only activated when default ranges are supplied? Should it not be a separate argument to the convert function whether to have those ops?

Reply:
> So now this is only activated when default ranges are supplied?

Correct

> Should it not be a separate argument to the convert function whether to have those ops?

In general yes, though I'd rather wait with implementing this until TF has proper support for this. See https://github.com/tensorflow/tensorflow/issues/38285